File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/actprm.js
export default function area() {
  return {
    current: 1,
    size: 20,
    isNewRecord: false,
    query: {
      // 表id编号
      actPrmId: undefined,
      // 活动类别编号
      code: undefined,
      // 活动类别名称
      name: undefined,
      // 备注
      apDesc: undefined,
      // 状态取自参数表
      stat: undefined,
      // 显示顺序
      sort: undefined,
      hideDeptId: undefined,
      dataModeId: undefined,
      parentCoderId: undefined,
      selectorModeId: undefined,
      conditionTypeId: undefined,
      // 所属系统ID
      sysId: undefined,
      // 预留字段1
      reserveCokumn1: undefined,
      // 预留字段2
      reserveCokumn2: undefined,
      // 预留字段3
      reserveCokumn3: undefined,
      // 日期类型（当DATA_MODE为DATE时的扩展）
      dateType: undefined,
      // 试算是否显示字段，0显示1不显示
      trialShow: undefined,
      // 隐藏预留字段
      parentValue: undefined,
      // 转换表达式
      value: undefined
    },
    rules: {
      actPrmId: [{
        required: true,
        message: '请输入表id编号',
        trigger: 'blur'
      }],
      code: [{
        required: true,
        message: '请输入活动类别编号',
        trigger: 'blur'
      }],
      name: [{
        required: true,
        message: '请输入活动类别名称',
        trigger: 'blur'
      }],
      apDesc: [{
        required: true,
        message: '请输入备注',
        trigger: 'blur'
      }],
      stat: [{
        required: true,
        message: '请输入状态取自参数表',
        trigger: 'blur'
      }],
      sort: [{
        required: true,
        message: '请输入显示顺序',
        trigger: 'blur'
      }],
      hideDeptId: [{
        required: true,
        message: '请输入',
        trigger: 'blur'
      }],
      dataModeId: [{
        required: true,
        message: '请输入',
        trigger: 'blur'
      }],
      parentCoderId: [{
        required: true,
        message: '请输入',
        trigger: 'blur'
      }],
      selectorModeId: [{
        required: true,
        message: '请输入',
        trigger: 'blur'
      }],
      conditionTypeId: [{
        required: true,
        message: '请输入',
        trigger: 'blur'
      }],
      sysId: [{
        required: true,
        message: '请输入所属系统ID',
        trigger: 'blur'
      }],
      reserveCokumn1: [{
        required: true,
        message: '请输入预留字段1',
        trigger: 'blur'
      }],
      reserveCokumn2: [{
        required: true,
        message: '请输入预留字段2',
        trigger: 'blur'
      }],
      reserveCokumn3: [{
        required: true,
        message: '请输入预留字段3',
        trigger: 'blur'
      }],
      dateType: [{
        required: true,
        message: '请输入日期类型（当DATA_MODE为DATE时的扩展）',
        trigger: 'blur'
      }],
      trialShow: [{
        required: true,
        message: '请输入试算是否显示字段，0显示1不显示',
        trigger: 'blur'
      }],
      parentValue: [{
        required: true,
        message: '请输入隐藏预留字段',
        trigger: 'blur'
      }],
      value: [{
        required: true,
        message: '请输入转换表达式',
        trigger: 'blur'
      }]
    }
  }
}

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/ActPrm.java
package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.FieldFill;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import org.springclouddev.core.mp.base.BaseEntity;

/**
 * 系统活动参数表实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_act_prm")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "ActPrm对象", description = "系统活动参数表")
public class ActPrm extends BaseEntity {

    private static final long serialVersionUID = 1L;

    /**
     * 表id编号
     */
    @ApiModelProperty(value = "表id编号")
    @TableId("act_prm_ID")
    private Long id;
    /**
     * 活动类别编号
     */
    @ApiModelProperty(value = "活动类别编号")
    @TableField("CODE")
    private String code;
    /**
     * 活动类别名称
     */
    @ApiModelProperty(value = "活动类别名称")
    @TableField("NAME")
    private String name;
    /**
     * 备注
     */
    @ApiModelProperty(value = "备注")
    @TableField("AP_DESC")
    private String apDesc;
    /**
     * 状态取自参数表
     */
    @ApiModelProperty(value = "状态取自参数表")
    private String stat;
    /**
     * 显示顺序
     */
    @ApiModelProperty(value = "显示顺序")
    @TableField(fill = FieldFill.INSERT)
    private Integer sort;
    @TableField("HIDE_DEPT_ID")
    private String hideDeptId;
    @TableField("DATA_MODE_ID")
    private String dataModeId;
    @TableField("PARENT_CODER_ID")
    private String parentCoderId;
    @TableField("SELECTOR_MODE_ID")
    private String selectorModeId;
    @TableField("CONDITION_TYPE_ID")
    private String conditionTypeId;
    /**
     * 所属系统ID
     */
    @ApiModelProperty(value = "所属系统ID")
    @TableField("SYS_ID")
    private String sysId;
    /**
     * 预留字段1
     */
    @ApiModelProperty(value = "预留字段1")
    private String reserveCokumn1;
    /**
     * 预留字段2
     */
    @ApiModelProperty(value = "预留字段2")
    private String reserveCokumn2;
    /**
     * 预留字段3
     */
    @ApiModelProperty(value = "预留字段3")
    private String reserveCokumn3;
    /**
     * 日期类型（当DATA_MODE为DATE时的扩展）
     */
    @ApiModelProperty(value = "日期类型（当DATA_MODE为DATE时的扩展）")
    @TableField("DATE_TYPE")
    private String dateType;
    /**
     * 试算是否显示字段，0显示1不显示
     */
    @ApiModelProperty(value = "试算是否显示字段，0显示1不显示")
    private Integer trialShow;
    /**
     * 隐藏预留字段
     */
    @ApiModelProperty(value = "隐藏预留字段")
    private String parentValue;
    /**
     * 转换表达式
     */
    @ApiModelProperty(value = "转换表达式")
    private String value;


}
